src/node_dotenv.cc

```diff
@@ -303,7 +391,8 @@ void Dotenv::ParseContent(const std::string_view input) {
   }
 }
 
-Dotenv::ParseResult Dotenv::ParsePath(const std::string_view path) {
+Dotenv::ParseResult Dotenv::ParsePath(const std::string_view path,
+                                      const std::set<std::string> sections) {
```

Comment:
If you don't add &, you'll always copy.

```suggestion
                                      const std::set<std::string>& sections) {
```